code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -185,3 +218,138 @@
 	if(!can_use_ability(target, override_flags = XACT_IGNORE_SELECTED_ABILITY))
 		return ..()
 	return TRUE
+
+
+
+// ***************************************
+// *********** Rage
+// ***************************************
+/datum/action/xeno_action/activable/rage
+	name = "Rage"
+	action_icon_state = "rage"
+	mechanics_text = "Use while at 50% health or lower to gain extra slash damage, resistances and speed in proportion to your missing hit points. This bonus is increased and you regain plasma while your HP is negative."
+	ability_name = "Rage"
+	plasma_cost = 0 //We're limited by cooldowns, not plasma
+	cooldown_timer = 120 SECONDS
+	keybind_flags = XACT_KEYBIND_USE_ABILITY | XACT_IGNORE_SELECTED_ABILITY
+	keybind_signal = COMSIG_XENOABILITY_RAGE
+	var/rage_power
+
+/datum/action/xeno_action/activable/rage/on_cooldown_finish()
+	to_chat(owner, "<span class='xenodanger'>We are able to enter our rage once again.</span>")
+	owner.playsound_local(owner, 'sound/effects/xeno_newlarva.ogg', 25, 0, 1)
+	return ..()
+
+/datum/action/xeno_action/activable/rage/can_use_ability(atom/A, silent = FALSE, override_flags)
+	. = ..()
+
+	var/mob/living/carbon/xenomorph/ravager/rager = owner
+
+	var/rage_health_threshold = rager.maxHealth * 0.5 //Need to be at 50% of max hp or lower to rage
+	if(rager.health > rage_health_threshold)
+		if(!silent)
+			to_chat(rager, "<span class='xenodanger'>Our health isn't low enough to rage! We must take [rager.health - rage_health_threshold] more damage!</span>")
+		return FALSE
+
+
+/datum/action/xeno_action/activable/rage/use_ability(atom/A)
+	var/mob/living/carbon/xenomorph/ravager/X = owner
+
+	rage_power = (1-(X.health/X.maxHealth)) * 0.5 //Calculate the power of our rage; scales with difference between current and max HP
+
+	if(X.health < 0) //Double dip on the negative HP; gain + 0.005 rage power per point of negative HP
+		rage_power += X.health * -0.005
+
+	rage_power = min(1, rage_power) //Cap rage power so that we don't get way too insane.
+
+	var/rage_power_radius = CEILING(rage_power * 6, 1) //Define radius of the SFX
+
+	X.visible_message("<span class='danger'>\The [X] becomes frenzied, bellowing with a shuddering roar!</span>", \
+	"<span class='highdanger'>We bellow as our fury overtakes us! RIP AND TEAR!</span>")
+	X.do_jitter_animation(1000)
+
+
+	//Roar SFX; volume scales with rage
+	playsound(X.loc, 'sound/voice/alien_roar2.ogg', clamp(100 * rage_power, 25, 80), 0)
+
+	var/bonus_duration
+	if(rage_power > 0.5) //If we're super pissed it's time to get crazy
+		bonus_duration = 1 SECONDS //Compensation for the obscuring visual effect
+		var/obj/screen/plane_master/floor/OT = locate(/obj/screen/plane_master/floor) in X.client.screen
+		OT.add_filter("rage_outcry", 2, list("type" = "radial_blur", "size" = 0.15))
+		addtimer(CALLBACK(OT, /atom.proc/remove_filter, "rage_outcry"), bonus_duration)
+		var/obj/screen/plane_master/game_world/GW = locate(/obj/screen/plane_master/game_world) in X.client.screen
```

Comment:
Try using the hud used managed planes instead:
`		var/obj/screen/plane_master/floor/OT = victim.hud_used.plane_masters["[FLOOR_PLANE]"]`
The locate was something I did while testing
Also you might want to consider motion blur filter here instead to indicate narrowed vision better

Reply:
I tried the motion blur here; it looked pretty unimpressive unless you've got some good values to share.